shell/browser/window_list.cc

```diff
@@ -81,19 +93,27 @@ void WindowList::RemoveObserver(WindowListObserver* observer) {
 // static
 void WindowList::CloseAllWindows() {
   WindowVector windows = GetInstance()->windows_;
+  std::vector<base::WeakPtr<NativeWindow>> weak_windows =
+      ConvertToWeakPtrVector(windows);
```

Comment:
`WindowVector windows = GetInstance()->windows_` is doing a copy of vector, can just pass `ConvertToWeakPtrVector(GetInstance()->windows_)` here to save the copy.